src/components/services/Ping.vue

```diff
@@ -45,6 +46,8 @@ export default {
 .status {
   font-size: 0.8rem;
   color: var(--text-title);
+  white-space: nowrap;
+  margin-left: 0.25rem;
```

Comment:
Unrelated to the method support ? what does it changes ? If you could include a screenshot it would be awesome.

Reply:
When the column is narrow the colored circle and text can appear on separate lines as seen at  https://imgur.com/a/rFmjJLe This change ensures that line break doesn't happen so the circle is always to the left of the text.